server/graphql/v2/mutation/OrderMutations.js

```diff
@@ -177,6 +186,35 @@ const orderMutations = {
         order = await order.update({ PaymentMethodId: newPaymentMethod.id });
       }
 
+      // amount
+      if (amount !== order.totalAmount) {
+        if (amount < 100) {
+          throw new Error('Invalid amount.');
+        }
```

Comment:
Edge case to consider: what's happening if "fees on top" are attached to the order?